### PR TITLE
Port to project.json/xproj and add .NET Standard support

### PR DESCRIPTION
**Pull request by jeremymeng**

### CODE REVIEW

**Comment by jeremymeng** [comment]
We are working with a customer who is using this package so I am submitting this PR.

File: Build.ps1 [hunk 1]
```diff
@@ -1,85 +1,15 @@
-param(
-    [String] $majorMinor = "0.0",  # 2.0
-    [String] $patch = "0",         # $env:APPVEYOR_BUILD_VERSION
-    [String] $customLogger = "",   # C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll
-    [Switch] $notouch,
-    [String] $sln                  # e.g serilog-sink-name
-)
+Push-Location $PSScriptRoot
```
Comment (nblumhardt) [inline Build.ps1]: Can you please replace this with the Build.ps1 from serilog/serilog? Thanks!

File: src/Serilog.Sinks.Logentries/project.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+﻿{
+  "version": "2.0.0",
```
Comment (nblumhardt) [inline src/Serilog.Sinks.Logentries/project.json]: 2.x of this sink is already published, can we use 3.0.0-* please?

**Comment by nblumhardt** [comment]
Looks great, couple of minor comments but good to go otherwise. Thanks!

**Comment by nblumhardt** [comment]
:+1: thanks!

**Pull request merged**